superset/assets/javascripts/dashboard/components/SaveModal.jsx

```diff
@@ -13,6 +13,9 @@ const propTypes = {
   css: PropTypes.string,
   dashboard: PropTypes.object.isRequired,
   triggerNode: PropTypes.node.isRequired,
+  readFilters: PropTypes.func,
```

Comment:
you should add defaults or the calls below could throw

Reply:
fixed.